### ISSUE TITLE
404 error on conclusion when packaged with Maven

### ISSUE THREAD

**Issue by LeMikaelF** [LeMikaelF/recitalinteractif#6]
Jetty can't find static files when packaged with `mvn package`.

```
HTTP ERROR 404

Problem accessing /conclusion/conclusion.html. Reason:

    Not Found
```

**Comment by LeMikaelF** [comment]
static html also not served; it looks like the only server functionality is the `/webjars` endpoint. Also occurs when jar is packaged by Intellij. Maybe resources aren't being loaded correctly?

**Comment by LeMikaelF** [comment]
Solved in #8
